lesson05/assignment/index.html

```diff
@@ -1,15 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="css/style.css">
         <title>Assignment 05 - CSS Grid</title>
     </head>
     <body>
+        
+        <!-- Setting up simple html layout for flex boxes -->
+
+        <header class="header">
+            <p>Header</p>
+        </header>
+
+        <!-- using BEM in nav to simplify the nested items in scss-->
+
+        <div class="nav">
```

Comment:
I would suggest using a nav tag here instead of a div.  If you use the class .nav for styling, then you can add it to the nav tag like <nav class="nav">